aces/Loris
# [Conflict Resolver] Filters: displaying user projects and sites

This PR:
1. Migrates the filter helpers from \DataFrameworkMenu to \Utility and add an $allsitesPermission parameter in getSiteOptions
2. Modifies the way project/subproject/site filters are populated. Now those filters are displayed with values the current user has permission for.
3. Fixes the integration tests

## To test
* Visit the module and check that all data entries belong to sites and projects the user has access to.
* Visit the module and check that the site/project/subproject menu filters contain entries the user has access to.

Related to #6829
Related to #6729
Replaces #7535
